onset_maproom/tests/test_calc.py

```diff
@@ -5,6 +5,55 @@
 import data_test_calc
 
 
+def test_longest_run_length():
```

Comment:
I retract this comment. There's a way to specify the exact test name rather than a pattern that might match multiple tests:

```
python -m pytest tests/test_calc.py::test_longest_run_length
```